core/user_api.php

```diff
@@ -647,6 +649,8 @@ function user_delete_profiles( $p_user_id ) {
 function user_delete( $p_user_id ) {
 	$c_user_id = (int)$p_user_id;
 
+	event_signal( 'EVENT_MANAGE_USER_DELETE', array( $p_user_id ) );
```

Comment:
event is placed before user_ensure_unprotected()?
should be better placed after it?